docs/development/instrumentation/how-to-instrument-code.md

```diff
@@ -16,6 +16,24 @@ ctx, span := tracer.Start(ctx, "GET /tax")
 defer span.End()
 ```
 
+## How to Set Span Kind
+
+When creating a span, you must specify the span kind to indicate the relationship between the span and the operation.
```

Comment:
More generally, while I say RPC, mostly what we work with is RESTFul HTTP Services. We should provide guidance for both:

1. a GRPC Server / client, and 
2. A HTTP+JSON RESTFul server / client (e.g. OpenAPI)